torchaudio/transforms.py

```diff
@@ -699,6 +699,10 @@ class MuLawDecoding(torch.nn.Module):
 
     Args:
         quantization_channels (int, optional): Number of channels. (Default: ``256``)
+    Example
```

Comment:
Let's add a line before ``Example``
```suggestion

    Example
```